app/elements/dashboard-business.html

```diff
@@ -29,7 +29,7 @@
 
     <firebase-document
       id='fire_business'
-      app-name="unicef-innovation-fund"
+      app-name="unicef-skygate-sandbox"
```

Comment:
same as above. In general we should avoid using app-name at all and just rely on the default one.